app/models.py

```diff
@@ -362,20 +373,27 @@ def is_expired(self):
         return self.expired < arrow.now()
 
 
-def generate_email() -> str:
-    """generate an email address that does not exist before"""
-    random_email = random_words() + "@" + EMAIL_DOMAIN
+def generate_email(scheme: int = 1, in_hex: bool = False) -> str:
```

Comment:
`scheme: int = 1` could be replaced by `scheme: int = AliasGeneratorEnum.word`. The reason for that is enum should be used instead of its value.